src/store/createStore.ts

```diff
@@ -11,14 +11,14 @@ export interface Store {
   // Account
   account: {
     value: string;
-    nestedObject: {
-      nestedValue1: string;
-      nestedValue2: number;
+    keypair: {
+      publicKey: string;
+      secretKey: string;
```

Comment:
I don't think we should save the secret key in the store. We will only need the public key for the Friendbot, right?

Reply:
I thought it would be nice to save it so users can use it when they make transactions on `TESTNET/FUTURENET`. What do you think?